staging/vhost-device-sound/Cargo.toml

```diff
@@ -6,6 +6,7 @@ description = "A virtio-sound device using the vhost-user protocol."
 repository = "https://github.com/rust-vmm/vhost-device"
 readme = "README.md"
 keywords = ["vhost", "sound", "virtio-sound", "virtio-snd", "virtio"]
+categories = ["multimedia::audio"]
```

Comment:
What about adding also `virtualization`?

Reply:
I missed the most obvious one in the category slug list :/ thanks! 😃

Reply:
Let's fix in another PR ;-)

Reply:
#517